dot/network/gossip_test.go

```diff
@@ -42,7 +45,7 @@ func TestGossip(t *testing.T) {
 
 	nodeA := createTestService(t, configA)
 	handlerA := newTestStreamHandler(testBlockAnnounceMessageDecoder)
-	nodeA.host.registerStreamHandler("", handlerA.handleStream)
+	nodeA.host.registerStreamHandler(protocolToTest, handlerA.handleStream)
```

Comment:
just wondering why it needs a different protocol?

Reply:
I've updated to a non-empty value just for test but now I put the original `""` value